rita/src/client.rs

```diff
@@ -210,6 +210,7 @@ fn main() {
             .route("/wifi_settings/mesh", Method::POST, set_wifi_mesh)
             .route("/info", Method::GET, get_own_info)
             .route("/version", Method::GET, version)
+            .route("/wipe", Method::POST, wipe)
```

Comment:
Make the method DELETE, it better fits the purpose and requires an extra step which should help ensure that the person making the request truly knows what they want.

Reply:
Hmm... Didn't think of it. Seems like good fit. Speaking of semantics that would be more appropriate to call `DELETE /wireguard`...